src/three/math/TileBoundingVolume.js

```diff
@@ -73,10 +66,9 @@ export class TileBoundingVolume {
 		if ( obb ) {
 
 			// the obb transform contains no scale
-			_ray.copy( ray ).applyMatrix4( obb.inverseTransform );
-			if ( _ray.intersectBox( obb.box, _obbVec ) ) {
+			if ( obb.intersectsRay( ray, _obbVec ) ) {
```

Comment:
Shall be `intersectRay`, `intersectsRay` only have one parameter and  thus `_obbVec` components will never be assigned.

```suggestion
			if ( obb.intersectRay( ray, _obbVec ) ) {
```

Reply:
Oops, thank you!